packages/paste-core/layout/flex/src/types.tsx

```diff
@@ -18,7 +18,7 @@ export type Basis = ResponsiveValue<BasisOptions>;
 export type WrapOptions = boolean;
 export type Wrap = ResponsiveValue<WrapOptions>;
 
-export interface FlexProps extends LayoutProps, MarginProps, PaddingProps {
+export interface FlexProps extends Omit<LayoutProps, 'verticalAlign'>, MarginProps, PaddingProps {
```

Comment:
What is it that Omit is doing here? I don't think I've ever seen it used before.

Reply:
Good question! Because the Flex component provides it's own `vertical` and `vAlignContent` props, it doesn't make sense to allow you to use the Box style prop `verticalAlign` as they would potentially clash.

So I want all the layout style props, but I don't want `verticalAlign`.

Reply:
Ah okay, so bc of that clash, we're saying that: from LayoutProps, omit the 'verticalAlign' one. Got it!